### ISSUE TITLE
Overflow menu behind statusbar

### ISSUE THREAD

**Issue by Guuz** [vonovak/react-navigation-header-buttons#103]
Hi!

Is it possible to apply insets to the overflow menu? Internally this is done in the Provider (i looked at the source) but i don't see a way to change it. For React Native it is set to 0 (https://github.com/vonovak/react-navigation-header-buttons/blob/master/src/overflowMenu/statusBarUtils.js). But this is not always the case. If you render your app edge-to-edge, as is strongly recommended by Google itself, you get this issue:
![image](https://user-images.githubusercontent.com/810327/99688484-259d9280-2a86-11eb-929e-c545ffba1732.png)

**Comment by vonovak** [comment]
hello and thanks for reporting!

> if you render your app edge-to-edge, as is strongly recommended by Google itself, you get this issue:


how do you go about that edge-to-edge rendering? I haven't tested this scenario which is why it doesn't work. Let me know how to do this and I would fix it (no timeline though). Otherwise feel free to open a PR. Thanks!

**Comment by Guuz** [comment]
@vonovak Not sure if I'm forgetting anything. It's been a while. But I think it's only 2 or 3 steps:

**MainActivity onCreate()**
```
window.decorView.findViewById<View>(android.R.id.content).systemUiVisibility = View.SYSTEM_UI_FLAG_LAYOUT_HIDE_NAVIGATION or View.SYSTEM_UI_FLAG_LAYOUT_STABLE
```

**High in the RN App code where the app starts**
```
<StatusBar backgroundColor="transparent" animated /> // Animated is not required
```

**styles.xml**
```
<item name="android:statusBarColor">@android:color/transparent</item>
<item name="android:navigationBarColor">@android:color/transparent</item> // This is the important one for the system navigation bar
// This is enough for Android 29 and higher. You need a little extra for below.
```

**Comment by vonovak** [comment]
I'm sorry, been a little too busy. If anyone has time for this, I'd accept a PR. Thanks

**Comment by dvbeelen** [comment]
Hi @vonovak! 

I've set up a PR that should resolve this issue: https://github.com/vonovak/react-navigation-header-buttons/pull/110

**Comment by vonovak** [comment]
closed by #110
